package/gluon-iptables-clamp-mss-to-pmtu/files/lib/gluon/mesh-vpn/iptables-mss.rules

```diff
@@ -1,3 +1,3 @@
 *mangle
--A FORWARD -o mesh-vpn+ -p tcp -m tcp --tcp-flags SYN,RST SYN -j TCPMSS --clamp-mss-to-pmtu
+-A FORWARD -o vpn+ -p tcp -m tcp --tcp-flags SYN,RST SYN -j TCPMSS --clamp-mss-to-pmtu
```

Comment:
`vpn` is not very descriptive. I don't think we should change `mesh-vpn`, let's fix the script instead.

Reply:
Done.